src/util/Recoil_batch.js

```diff
@@ -0,0 +1,15 @@
+function defaultBatch() {
+  throw new Error('No batching function specified.');
+}
+
+let batch = defaultBatch;
+
+// Allow injecting another batching function later
+const setBatch = newBatch => (batch = newBatch);
+
+const batchUpdates = callback => batch(callback);
+
+module.exports = {
+  setBatch,
```

Comment:
Since this is setting up the batcher instead of an individual batch, how about `setBatcher()`?